pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -37,23 +41,24 @@ type dataFeedsAggregator struct {
 
 var _ types.Aggregator = (*dataFeedsAggregator)(nil)
 
-// EncodableOutcome is a list of AggregatedPricePoints
+// This Aggregator has two phases:
+//  1. Agree on valid trigger signers by extracting them from event metadata and aggregating using MODE (i.e. F+1 copies needed).
```

Comment:
what is the comment about F+1? is that specific to MODE? if so, i don't understand how one is related to the other

Reply:
yeah, MODE but the value needs to appear at least F+1 times